source/MaterialXGenShader/Util.h

```diff
@@ -101,9 +101,15 @@ vector<Vector2> getUdimCoordinates(const StringVec& udimIdentifiers);
 /// 0..1 space.
 void getUdimScaleAndOffset(const vector<Vector2>& udimCoordinates, Vector2& scaleUV, Vector2& offsetUV);
 
-/// Checks if graph output requires normals to be transformed from tangent 
-/// space to world space.
-bool connectsToNormalMapNode(OutputPtr output);
+/// Check if a output is connected to nodes of a given category.
+/// \param Return the node if found.
```

Comment:
These should be "@param" for consistency with other Doxygen comments in the code.